clusterloader2/pkg/measurement/common/profile.go

```diff
@@ -187,6 +187,11 @@ func (p *profileMeasurement) gatherProfile(c clientset.Interface) ([]measurement
 	for _, host := range p.config.hosts {
 		profilePrefix := fmt.Sprintf("%s_%s_%s", host, p.config.componentName, p.name)
 
+		// ssh to collect profile metrics not supported on aks masters
+		if p.config.provider == "aks" {
+			return nil, nil
```

Comment:
It's possible to do it without SSH, see line 202. Would it be feasible for aks?

Reply:
testing this out, I think it should work...will report back